src/core/load-fragment.c

```diff
@@ -1335,6 +1335,12 @@ int config_parse_exec_cpu_affinity(const char *unit,
         assert(rvalue);
         assert(data);
 
+        if (streq(rvalue, "numa")) {
+                c->numa_policy.derive_cpu_affinity = true;
+                return 0;
+        } else if (isempty(rvalue))
+                c->numa_policy.derive_cpu_affinity = false;
+
         return parse_cpu_set_extend(rvalue, &c->cpu_set, true, unit, filename, line, lvalue);
```

Comment:
Sorry, but I don't think this is good UI.
If I do
```
CPUAffinity=1-3
CPUAffinity=numa
CPUAffinity=5-7
```
having the middle option win is not expected.

We decided that it is OK to have "numa" cancel out any specific list of CPUs, but please make it symmetrical, and setting a list should cancel out "numa" again.